packages/api/resolvers/mutations/accounts/loginAsGuest.js

```diff
@@ -1,10 +1,17 @@
 import { log } from 'meteor/unchained:core-logger';
-import callMethod from '../../../callMethod';
+import { accountsServer } from 'meteor/unchained:core-accountsjs';
 
-export default function loginAsGuest(root, methodArguments, context) {
+export default async function loginAsGuest(root, methodArguments, context) {
   log('mutation loginAsGuest');
-  return callMethod(context, 'login', {
-    ...methodArguments,
-    createGuest: true,
-  });
+  const { user: id, token } = await accountsServer.loginWithService(
```

Comment:
In the loginWithPassword resolver you use evaluateContext and here you don't?

Reply:
This totally depends on whether the context was needed in later operations (which we did in `LoginSuccess` hook) and if method allows for passing the context.

https://github.com/accounts-js/accounts/blob/master/packages/server/src/accounts-server.ts#L176

Though the second condition of the possibility is much weaker since if there's a use case for a hook then I'd extend the methods.